src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    to set `log.to` to file. Otherwise, `log.file` is ignored.
+]])
+
+M['log.format'] = format_text([[
+    Specify a format that is used for a log entry. The following formats
+    are supported:
+
+    - `plain`: a log entry is formatted as plain text.
+    - `json`: a log entry is formatted as JSON and includes additional fields.
+]])
+
+M['log.level'] = format_text([[
+    Specify the level of detail logs have. There are the following levels:
+
+    - 0: `fatal`
+    - 1: `syserror`
+    - 2: `error`
+    - 3: `crit`
+    - 4: `warn`
+    - 5: `info`
+    - 6: `verbose`
+    - 7: `debug`
+
+    By setting log.level, you can enable logging of all events with
+    severities above or equal to the given level.
+]])
+
+M['log.modules'] = format_text([[
+    Configure the specified log levels (`log.level`) for different modules.
+
+    You can specify a logging level for the following module types:
+
+    - Modules (files) that use the default logger. Example: Set log levels for
+    files that use the default logger.
+    - Modules that use custom loggers created using the `log.new()` function.
+    Example: Set log levels for modules that use custom loggers.
+    - The tarantool module that enables you to configure the logging level
+    for Tarantool core messages. Specifically, it configures the logging level
+    for messages logged from non-Lua code, including C modules. Example: Set
+    a log level for C modules.
+]])
+
+M['log.modules.*'] = format_text([[
+    Path to module.
+
+    For example: you have module placed by the following path:
+    `test/module.lua.` To configure logging levels, you need to
+    provide module names corresponding to paths to these modules:
+    ``` test.module: 'verbose' ```.
+]])
+
+M['log.nonblock'] = format_text([[
+    Specify the logging behavior if the system is not ready to write.
+    If set to `true`, Tarantool does not block during logging if the system
+    is non-writable and writes a message instead. Using this value may
+    improve logging performance at the cost of losing some log messages.
+]])
+
+M['log.pipe'] = format_text([[
+    Start a program and write logs to its standard input (`stdin`). To send
+    logs to a program's standard input, you need to set `log.to` to `pipe`.
+]])
+
+M['log.syslog'] = 'Syslog configurations params.'
+
+M['log.syslog.facility'] = format_text([[
+    Specify the syslog facility to be used when syslog is enabled.
+    To write logs to syslog, you need to set `log.to` to `syslog`.
+]])
+
+M['log.syslog.identity'] = format_text([[
+    Specify an application name used to identify Tarantool messages in
+    syslog logs. To write logs to syslog, you need to set `log.to` to `syslog`.
+]])
+
+M['log.syslog.server'] = format_text([[
+    Set a location of a syslog server. This option accepts one of the following
+    values:
+
+    - An IPv4 address. Example: `127.0.0.1:514`.
+    - A Unix socket path starting with unix:. Examples:
```

Comment:
It may be unclear, whether the period in `unix:.` is a punctuation mark or part of the format. I would use backticks to make it clear.

Reply:
AFAIR, it is from the `iproto.advertise.client` description. The highlighted problem is fixed now.

But I would say 'a TCP socket address' instead of just `an address` -- it seems too general.

Reply:
I prefer `an address`.
```lua
I['log.syslog.server'] = format_text([[
    Set a location of a syslog server. This option accepts one of the following
    values:

    - An address. Example: `127.0.0.1:514`.
    - A Unix socket path starting with `unix:`.
      Examples: `unix:/dev/log` on Linux or `unix:/var/run/syslog` on macOS.

    To write logs to syslog, you need to set `log.to` to `syslog`.
]])
```

Reply:
I think it is too general, but OK.